.changeset/honest-experts-battle.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/design-system-components": patch
+---
+
+`AppSideNav` - fixed bug where scrolling was blocked when the `AppSideNav` was expanded on desktop views. Also fixed bug where able to focus links that are visually hidden.
```

Comment:
```suggestion
`AppSideNav` - Fixed bug where scrolling was blocked when the `AppSideNav` was expanded on desktop views. Also fixed bug which allowed user to focus links that were visually hidden.
```

Reply:
Some of the wording above seemed a little awkward so I edited to try to clarify. (Feel free to tweak further.)

Reply:
This is definitely better!